Gemfile

```diff
@@ -6,7 +6,7 @@ gem "gds-sso"
 gem "govuk_app_config"
 gem "plek"
 
-gem "mongo"
+gem "mongo", "~> 2.15.1"
```

Comment:
Yikes. Guessing we have a card tracking our need to upgrade Mongo?

Reply:
There's a tech-debt card: https://trello.com/c/lSpntlfk/81-mongo-26-has-reached-end-of-life

...but it's a bit broad. I started working on it on and off when router/router-api/authenticating-proxy/imminence were all shared in common by F&V. Imminence is a dead end, but authenticating proxy is well under way (I need to merge a few PRs and then schedule a time to do a switchover in integration). I've done a lot of the work on router to upgrade it from Mongo 2.6 to where it can use a modern version of Mongo or DocumentDB, but I wanted to get AP switched over first. If anyone from N&P fancies helping out, talk to me.

(There's also the whole question of whether this eminently relational data should even be in Mongo in the first place)

Reply:
We have cards in our Planning board that capture this and I talked to Kelvin about it.
I want to consider switching to a relational DB.